charts/soroban-rpc/templates/soroban-rpc-cm.yaml

```diff
@@ -0,0 +1,212 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "common.fullname" . }}
+  {{- if .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels:
+    app: {{ template "common.fullname" . }}
+    chart: {{ template "common.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  soroban-rpc.cfg: |
+    ENDPOINT={{ .Values.sorobanRpc.sorobanRpcConfig.endpoint | default "0.0.0.0:8000" | quote }}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).friendbotUrl -}}
+    FRIENDBOT_URL={{ .Values.sorobanRpc.sorobanRpcConfig.friendbotUrl | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).networkPassphrase -}}
+    NETWORK_PASSPHRASE={{ .Values.sorobanRpc.sorobanRpcConfig.networkPassphrase | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).stellarCoreUrl -}}
+    STELLAR_CORE_URL={{ .Values.sorobanRpc.sorobanRpcConfig.stellarCoreUrl | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).captiveCoreConfigPath -}}
+    CAPTIVE_CORE_CONFIG_PATH={{ .Values.sorobanRpc.sorobanRpcConfig.captiveCoreConfigPath | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).captiveCoreStoragePath -}}
+    CAPTIVE_CORE_STORAGE_PATH={{ .Values.sorobanRpc.sorobanRpcConfig.captiveCoreStoragePath | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).captiveCoreUseDb -}}
+    CAPTIVE_CORE_USE_DB={{ .Values.sorobanRpc.sorobanRpcConfig.captiveCoreUseDb }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).stellarCoreBinaryPath -}}
+    STELLAR_CORE_BINARY_PATH={{ .Values.sorobanRpc.sorobanRpcConfig.stellarCoreBinaryPath | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).historyArchiveUrls -}}
+    HISTORY_ARCHIVE_URLS={{ .Values.sorobanRpc.sorobanRpcConfig.historyArchiveUrls | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).dbPath -}}
+    DB_PATH={{ .Values.sorobanRpc.sorobanRpcConfig.dbPath | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).stellarCaptiveCorePort -}}
+    STELLAR_CAPTIVE_CORE_HTTP_PORT={{ .Values.sorobanRpc.sorobanRpcConfig.stellarCaptiveCorePort }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.sorobanRpcConfig).checkpointFrequence -}}
+    CHECKPOINT_FREQUENCY={{ .Values.sorobanRpc.sorobanRpcConfig.checkpointFrequence }}
+    {{ end }}
+  stellar-captive-core.cfg: |
+    {{ if (.Values.sorobanRpc.coreConfig).bucketDirPath -}}
+    BUCKET_DIR_PATH={{ .Values.sorobanRpc.coreConfig.bucketDirPath |quote -}}
+    {{ end }}
+    {{ if (.Values.sorobanRpc.coreConfig).database -}}
+    DATABASE={{ (.Values.sorobanRpc.coreConfig).database | quote }}
+    {{ end -}}
+    {{ if (.Values.sorobanRpc.coreConfig).experimentalBucketlistDb -}}
+    EXPERIMENTAL_BUCKETLIST_DB={{ (.Values.sorobanRpc.coreConfig).experimentalBucketlistDb }}
```

Comment:
for consideration, drop the 'experimental' part of these bucketlist parameter names, that way when the chart names don't have to change when the toml names change and drop that.

Reply:
Yeah it sounds good, I'll update the PR